File: solutions/javascript/resistor-color-duo/1/resistor-color-duo.js
export const decodedValue = (list) => {
  const COLORS = [
  'black',
  'brown',
  'red',
  'orange',
  'yellow',
  'green',
  'blue',
  'violet',
  'grey',
  'white',
  ]
  const answer = String(COLORS.indexOf(list[0])) + String(COLORS.indexOf(list[1]))
  return Number(answer)
};

File: solutions/python/resistor-color-duo/1/resistor_color_duo.py
def value(colors):
    answer = ""
    color_dict = {
        "black": '0',
        "brown": '1',
        "red": '2',
        "orange": '3',
        "yellow": '4',
        "green": '5',
        "blue": '6',
        "violet": '7',
        "grey": '8',
        "white": '9'
    }
    for index, c in enumerate(colors):
        if index > 1:
            continue
        answer += color_dict[c]
    return int(answer)
    
